tests/test_sageintacct/test_utils.py

```
@@ -1398,6 +1398,23 @@ def test_get_or_create_vendor(mocker, db):
     assert vendor.destination_id == 'non exiSting VENDOR iN intacct UsE aLl CaSeS'
 
 
+def test_get_or_create_vendor_with_none_sanitized_name(db):
```

Ruff (0.14.13)

Unused function argument: `db`

(ARG001)